twisted/python/test/test_zipstream.py

```diff
@@ -324,9 +324,9 @@ def _unzipIterChunkyTest(self, compression, chunksize, lower, upper):
         for r in uziter:
             pass
         self.assertEqual(r, 0)
-        newmd5 = md5(
-            tempdir.child("zipstreamjunk").open().read()).hexdigest()
-        self.assertEqual(newmd5, junkmd5)
+        with tempdir.child("zipstreamjunk").open() as f:
+            newmd5 = md5(f.read()).hexdigest()
+            self.assertEqual(newmd5, junkmd5)
```

Comment:
`self.assertEqual(newmd5, junkmd5)` can go outside of the context manager.  Maybe that's clearer?